### ISSUE TITLE
dry-run mode that is not changing existent files

### ISSUE THREAD

**Issue by e110c0** [GertjanReynaert/react-intl-translations-manager#98]
is there an easy way to make the report but not actually add the missing keys and output something like this:

```
Duplicate ids:

  No duplicate ids found, great!

Maintaining de.json:

Untranslated keys:
  login-form.input-placeholder: Your E-mail

Maintaining es.json:

Untranslated keys:
  login-form.input-placeholder: Your E-mail
  login-form.submit-button: Log in
```

That would be convenient if there is a more enhanced way of getting fallback languages or having an external tool for the translations.

It works when I overload most of the hooks and printers, but that's almost completely copying the original code.

**Comment by nightpool** [comment]
yeah, I agree that there's lots of situations where I don't want untranslated keys in the source files. We're using Weblate to translate our project, and it marks translations as either present or not present based on their status. Can this functionality be added?
